src/sandesh/First13TestCases/TestScenario_13.java

```diff
@@ -0,0 +1,80 @@
+/*program 1: count number of employee under each manager, verify count against properties file.
+output : 
+20209 -> 3
+20205 -> 3
+10101 -> 2
+*/
+
+package sandesh.First13TestCases;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Properties;
+import java.util.Set;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class TestScenario_13 {
+
+	private boolean compareWebAndPropertiesFile(WebDriver driver) throws IOException {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method.